pages/task/read/routers/view.js

```diff
@@ -88,6 +88,15 @@ module.exports = () => {
         .then(() => next())
         .catch(next);
     }
+    if (req.task.data.model === 'project' && req.task.data.action === 'update') {
+      const licenceHolderId = get(req.task, 'data.data.licenceHolderId');
+      return req.api(`/establishment/${req.establishmentId}/profile/${licenceHolderId}`)
+        .then(({ json: { data } }) => {
+          set(req.task, 'data.data.licenceHolder', data);
+        })
+        .then(() => next())
+        .catch(next);
+    }
```

Comment:
I'd be more inclined to do this in a workflow decorator and avoid the extra request round trip.

Reply:
fair.